modules/explorer/info.go

```diff
@@ -38,6 +42,32 @@ func (e *Explorer) BlockFacts(height types.BlockHeight) (modules.BlockFacts, boo
 	return bf.BlockFacts, true
 }
 
+// LatestBlockFacts returns a set of statistics about the blockchain as they appeared
+// at the latest block height in the explorer's consensus set.
+func (e *Explorer) LatestBlockFacts() modules.BlockFacts {
+	var bf blockFacts
```

Comment:
safety tells me that we should be grabbing a readlock on the explorer here, with a deferred unlock. Bolt has some special things that might make it unnecessary, but then you are assuming that all stateful accesses in this function are bolt, and that's cognitive overhead that's just invite race-condition and dead-lock based edge cases in the future.